Documentation/ListModule/UsingEffectively/Index.rst

```diff
@@ -1,6 +1,18 @@
 .. include:: /Includes.rst.txt
 .. _using-the-list-module-effectively:
 
+============
+Introduction
+============
+
+The list module allows you to browse through
+each page or folder within your site and view all of the records that
+are stored within it. The List Module also gives you the ability to
+create and manage records that don't have a dedicated Module.
+
+Sub pages, content elements and news stories are examples of the different
```

Comment:
Hmm, are we really talking about "news stories"? Not about "news articles" or "news records"?

Reply:
Yes